.claude/commands/refactor.md

```
@@ -0,0 +1,127 @@
+---
+description: Execute TDD Refactor Phase - improve code structure while keeping tests green
+argument-hint: <refactoring description>
+---
+
+**User arguments:**
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)